dispatcher/cli/dcos_spark/cli.py

```diff
@@ -8,6 +8,8 @@
     dcos spark run --help
     dcos spark run --submit-args=<spark-args>
                    [--dcos-space=<dcos_space>]
+                   [--kerberos-principal=kerberos_principal]
+                   [--keytab-secret-path=<keytab_secret>]
```

Comment:
@ArtRand There is some mention of `--tgt` in this custom PR: https://github.com/mesosphere/spark/commit/40b05d3f2321c366a070f5894e95d74b074c89c4